Unix/http/http.c

```diff
@@ -838,13 +854,31 @@ static Http_CallbackResult _WriteHeader(
     if (!handler->ssl)
     {
 #if ENCRYPT_DECRYPT
+        Page *pOldPage = handler->sendPage;
+
         if (!Http_EncryptData(handler, (char**)&currentLine, &buf_size,  &handler->sendPage) ) {
              
             // If we fail it was an error. Not encrypting counts as failure Complain and bail
 
             trace_HTTP_EncryptionFailed();
             return PRT_RETURN_FALSE;
         }
+        else {
+            if (FORCE_TRACING || handler->enableTracing)
+            {
+                char before_encrypt[] = "\n------------ Before Encryption ---------------\n";
+                char before_encrypt_end[] = "\n------------ End Before ---------------\n";
+                _WriteTraceFile(ID_HTTPSENDTRACEFILE, &before_encrypt, sizeof(before_encrypt));
+                _WriteTraceFile(ID_HTTPSENDTRACEFILE, (char *)(pOldPage+1), pOldPage->u.s.size);
+                _WriteTraceFile(ID_HTTPSENDTRACEFILE, &before_encrypt_end, sizeof(before_encrypt_end));
+            }
+            /*
+            if (pOldPage != handler->sendPage)
+            {
+                PAL_Free(pOldPage);
+            }        
+            */
```

Comment:
I think this is now on the stack. Delete 875-880?

I consider these checkins to be permanent. Checked in code should be "clean". If you must have a commented out section (because you believe it will be needed later, and don't want to pull from git history), then have a comment as to WHY.